roles/platform/tasks/setup_azure_authz.yml

```diff
@@ -25,36 +25,66 @@
 
     - name: Remove Azure App as CDP Credential needs (re)creating
       when: plat__azure_xaccount_app_uuid is defined
-      command: "az ad app delete --id {{ plat__azure_xaccount_app_uuid }}"
+      block:
+        - name: Issue Azure App Delete command
+          command: "az ad app delete --id {{ plat__azure_xaccount_app_uuid }}"
+
+        - name: Wait for consistency
+          ansible.builtin.pause:
+            seconds: "{{ plat__azure_consistency_wait }}"
 
-    # Owner role is required for DWX
+    # Owner role is required for DWX if you are thinking of modifying this task
```

Comment:
Not anymore... I realize it's outside the scope of this PR, but DWX _just_ needs Contributor these days.

Reply:
Oooph. And single resource group. I'll create "Issues".